tools/clang/test/HLSLFileCheck/d3dreflect/empty_broadcasting_nodes.hlsl

```diff
@@ -4,8 +4,8 @@
 // CHECK:  StringBuffer (size = {{[0-9]+}} bytes)
```

Comment:
Should this one get `-Vd -validator-version 0.0` too?

Reply:
No, this tests the node shader data, which must be valid when targeting lib_6_8 (validator version 1.8 required).